specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -134,43 +197,138 @@
         }
       }
     },
-    "CanceledSubscriptionId": {
-      "description": "The ID of the canceled subscription",
+    "SubscriptionName": {
+      "type": "object",
+      "description": "The new name of the subscription.",
       "properties": {
-        "value": {
-          "description": "The ID of the canceled subscription",
+        "subscriptionName": {
           "type": "string",
-          "readOnly": true
+          "description": "New subscription name"
         }
       }
     },
-    "RenamedSubscriptionId": {
-      "description": "The ID of the subscriptions that is being renamed",
+    "planTypeEnum": {
+      "type": "string",
+      "description": "the planType.",
+      "enum": [
+        "basic_support",
+        "developer_support",
+        "standard_support",
+        "prodirect_support"
```

Comment:
What does 'prodirect' mean - will that be obvious to API users or does it need a description?